src/handlers/motions/motionStaked/motionStaked.ts

```diff
@@ -36,10 +39,54 @@ export default async (event: ContractEvent): Promise<void> => {
   if (stakedMotion) {
     const {
       id,
-      motionData: { usersStakes },
+      motionData: { usersStakes, events: updatedEvents },
       motionData,
     } = stakedMotion;
 
+    if (vote.eq(0) && !updatedEvents.find(({ name }) => name === 'ObjectionRaised')) {
```

Comment:
I think some of this logic could tidied up a little bit, e.g. 

``` js
const isFirstObjection = ...

if(isFirstObjection) {

}
```

instead of 

``` js
if (vote.eq(0) && !updatedEvents.find(({ name }) => name === 'ObjectionRaised')
```

But the main thing is please move all of this to a helper and follow the pattern in this file (e.g. as with user stakes)

``` js

 if (stakedMotion) {
    const {
      id,
      motionData: { usersStakes, events },
      motionData,
    } = stakedMotion;

const updatedEvents = getUpdatedEvents(events, vote, amount)

...

 await updateMotionInDB(id, {
      ...motionData,
      usersStakes: updatedUserStakes,
      motionStakes,
      remainingStakes,
      events: updatedEvents,
    });
```